src/components/Extension/CalDotComExtension/CalDotComExtension.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react'
+import { useTranslation } from 'next-i18next'
+
+import { ErrorPage } from '../../ErrorPage'
+import { BookAppointmentAction } from './BookAppointmentAction'
+
+import type {
+  Activity,
+  ExtensionActivityRecord,
+} from '../../../types/generated/types-orchestration'
```

Comment:
Same comment as for the `BookAppointmentAction` component.

Reply:
Good tip, thanks!